apps/passport-server/src/services/issuanceService.ts

```diff
@@ -872,6 +1039,81 @@ export class IssuanceService {
       return tickets;
     });
   }
+
+  public async verifyTicket(
```

Comment:
verify vs check ticket is unclear to me

Reply:
I've added an explanatory comment. "Verify" applies to all tickets apart from Devconnect, and "Check"/"CheckIn" applies only to Devconnect.